Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -522,9 +522,9 @@ void ChessWidget::playback_move(PlaybackDirection direction)
     update();
 }
 
-DeprecatedString ChessWidget::get_fen() const
+String ChessWidget::get_fen() const
```

Comment:
I don't think that this is a step in the correct direction.
If a function will ever return a normal `String` (I mean no SSO involved), we should return a `ErrorOr<String>`, not a `String`.

I think that we should properly convert this function and add the `.release_value_but_fixme_should_propagate_errors()` on the caller side.